app/controllers/users_controller.rb

```diff
@@ -14,6 +15,29 @@ def new
     @user = User.new
   end
 
+  def update
+    # @user = User.find(session[:user_id])
+    @user = User.find(params[:id])
```

Comment:
Seems like this might let you update a user other than yourself? Is that the desired behavior?